src/main/java/uk/gov/hmcts/reform/finrem/ccddatamigration/processor/DataMigrationProcessor.java

```diff
@@ -86,7 +86,7 @@ public void run(final String... args) {
             } else {
                 log.info("migrate multiple cases .....");
                 caseTypes.forEach(caseType -> {
-                    log.info("migrate  caseType .....", caseType);
+                    log.info("migrate caseType ....." + caseType);
```

Comment:
Using argument replacement like `"migrate caseType .....{}"` will give 1us performance gain per 100k invocations 😂 🙈

Reply:
this is the sort of comments we need more of, thanks @jakub-kozlowski 😂